backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/material/cockpit/filters/ProductFilterUtil.java

```diff
@@ -89,6 +87,18 @@ public static DocumentFilterDescriptor createFilterDescriptor()
 				.setWidgetType(DocumentFieldWidgetType.YesNo)
 				.setOperator(Operator.EQUAL);
 
+		final DocumentFilterParamDescriptor.Builder isActive = DocumentFilterParamDescriptor.builder()
+				.setFieldName(I_M_Product.COLUMNNAME_IsActive)
+				.setDisplayName(Services.get(IMsgBL.class).translatable(I_M_Product.COLUMNNAME_IsActive))
```

Comment:
1. either extract `Services.get(IMsgBL.class)` as a local var and use it in all 3 cases u have,
2. or use de.metas.i18n.TranslatableStrings#adElementOrMessage

Reply:
will do the first option, thx